Library/ViewModels/PledgePaymentPlansOptionViewModel.swift

```diff
@@ -1,13 +1,35 @@
+import Foundation
+import KsApi
 import ReactiveSwift
 
 public typealias PledgePaymentPlanOptionData = (
```

Comment:
1. If you need a model object here, it should be a `struct` and not a tuple. That way auto-complete and compile errors work significantly better.
2. This will eventually come from the API, correct? I assume this will be replaced by an API object, instead.

Reply:
I’ve updated this to use a `struct` as suggested. In fact, some of these models have already been converted to struct in another [PR](https://github.com/kickstarter/ios-oss/pull/2222). For point 2, only the `paymentIncrements` model will need to be updated to use the object from the API. The rest of the structure will remain as is for now.